internal/cutil/aliases.go

```diff
@@ -21,8 +21,14 @@ const (
 
 	// SizeTSize is the size of C.size_t
 	SizeTSize = C.sizeof_size_t
+
+	// IntSize is the size of C.int
+	IntSize = C.sizeof_int
 )
 
+// Compile-time assertion ensuring that Go's `int` is at least as large as C's.
+const _ = unsafe.Sizeof(int(0)) - unsafe.Sizeof(C.int(0))
```

Comment:
Could also be
```suggestion
const _ = unsafe.Sizeof(int(0)) - C.sizeof_int
```
but doesn't matter actuallly.

Reply:
That's probably nicer. Maybe later you / we could find a better place for these compile-time assertions, or is `cutil/aliases.go` ok?